data/examples/random_forest_regressor.py

```diff
@@ -0,0 +1,132 @@
+import csv
+import numpy as np
+from sklearn.ensemble import RandomForestRegressor
+import pandas as pd
+import scipy
+from sklearn.model_selection import KFold
+import os
+
+# Set random seed
+np.random.seed(0)
+
+# read interactions
+def read_interactions():
+    term_set = []
+    with open('./example-hierarchy-sets.tsv') as tsvfile:
+      reader = csv.reader(tsvfile, delimiter='\t')
+      for row in reader:
+        term_set.append(row)
+    return term_set
+
+# read genes
+def read_genes():
+    genes = []
+    with open('./example-gene-names.txt') as txtfile:
+      reader = csv.reader(txtfile, delimiter='\n')
+      for row in reader:
+        genes.extend(row)
+    return genes
+
+
+def pair_genes(genes, num_genes, gene_interactions):
+    # produce gene interaction pairs
+    gene_pairs = []
+    pair_scores = []
+    # class_dict = {"positive" : 1, "negative":2,"non-interaction":0}
+
+    for i in range(num_genes):
+        for j in range(i + 1, num_genes):
+            gene_pairs.append((genes[i], genes[j]))
+            pair_scores.append(gene_interactions[i][j])
+    return gene_pairs, pair_scores
+
+
+def build_geneterm_dict(genes, term_set):
+    # build gene dict which returns the terms that specific gene belongs
+    gene_dict = {}
+    for gene in genes:
+        for idx, term in enumerate(term_set):
+            if gene in term:
+                if gene not in gene_dict:
+                    gene_dict[gene] = [idx]
+                else:
+                    gene_dict[gene].append(idx)
+    return gene_dict
+
+def construct_feature_vector(gene_pairs, gene_dict, pair_scores):
+    feature_vector = np.zeros([4950, 100])  # 4950 x (99+1)  4950 gene combination(assume (g1,g2) is equivalent to (g2,g1))
```

Comment:
Not sure you want to hard-code the dimensionality, since it will be different for simulated/real data.